test/utils.js

```diff
@@ -28,9 +28,7 @@ function run(cmd, opts) {
         readLines(child, opts.onOutputLine);
     }
 
-    if (opts.onErrorLine) {
-        readLines(child, opts.onErrorLine, 'stderr');
-    }
+    readLines(child, (l) => { console.log(l); }, 'stderr');
```

Comment:
What was the reason for this change?

I see `onErrorLine` isn't used at all, but don't think we should get rid of it.

Reply:
@pgraham do you have some time to look into this? I can't wait for this change to happen ^^ ;)

Reply:
I had previously added this option to log any output on stderr for specific tests but decided it would be better to always do this and so removed the options since it was now unused. But I agree, it would be useful to keep so I've re-added it.